sk-python-hello-world/README.md

```diff
@@ -1 +1,40 @@
 # Semantic Kernel Python Hello World Starter
+
+The `sk-python-hello-world` console application demonstrates how to execute a semantic function.
+
+## Prerequisites
+
+- [Python](https://www.python.org/downloads/) 3.8 and above
+  - [Poetry](https://python-poetry.org/) is used for packaging and dependency management
+  - [Semantic Kernel Tools](https://marketplace.visualstudio.com/items?itemName=ms-semantic-kernel.semantic-kernel)
+
+## Configuring the starter
+
+The starter can be configured with a `.env` file in the project which holds api keys and other secrets and configurations.
+
+Make sure you have an
+[Open AI API Key](https://openai.com/api/) or
+[Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api)
+
+Copy those keys into the `.env` file:
```

Comment:
One thing we want to avoid is people accidentally checking in their API key.

There are a couple of patterns we are using to avoid this:

1. Creating example config files e.g. https://github.com/microsoft/semantic-kernel-starters/blob/main/sk-csharp-hello-world/Config/appsettings.json.azure-example and then adding the normal config file to the `.gitignore`
2. For .Net we use Secret Manager as an optional way to configure the endpoints, see https://github.com/microsoft/semantic-kernel-starters/blob/main/sk-csharp-hello-world/README.md#using-net-secret-manager

Can you make the equivalent changes

Reply:
You're right, that's how we were doing it in the semantic kernel as well.  Will update